client/components/mma/accountoverview/AccountOverview.tsx

```diff
@@ -119,7 +119,8 @@ const AccountOverviewRenderer = ([mdapiObject, cancelledProductsResponse]: [
 						</h2>
 						<Stack space={6}>
 							{activeProductsInCategory.map((productDetail) =>
-								featureSwitches.accountOverviewNewLayout ? (
+								featureSwitches.accountOverviewNewLayout ||
+								productDetail.tier === 'IAP' ? ( // ToDo: remove this check when moving to V2
```

Comment:
I decided not to try include IAPs in the old account overview card, not sure if this is the right approach